addons/sale_timesheet/tests/common.py

```diff
@@ -319,3 +319,20 @@ def setUpClass(cls, chart_template_ref=None):
             'taxes_id': False,
             'property_account_income_id': cls.account_sale.id,
         })
+
+        cls.service_prepaid = cls.env['product.product'].create({
+            'name': "Service delivered, create task in new project",
+            'standard_price': 10,
+            'list_price': 20,
+            'type': 'service',
+            'invoice_policy': 'delivery',
+            'uom_id': uom_hour.id,
+            'uom_po_id': uom_hour.id,
+            'default_code': 'SERV-DELI3',
+            'service_type': 'manual',
+            'service_tracking': 'task_in_project',
+            'project_id': False,  # will create a project
+            'taxes_id': False,
+            'property_account_income_id': cls.account_sale.id,
+            'service_policy': 'ordered_timesheet'
+        })
```

Comment:
It seems this product has the same configuration than the one in `cls.product_delivery_manual3` in the same file, isn't it ? 🤔

Reply:
This one has the 'service_policy' :  'ordered_timesheet'

Reply:
This field is a compute not stored one. So even you set a value for this field, the compute method is triggered to change to the correct  value. So, it is useless to set it. 🙂 
See [here](https://github.com/odoo/odoo/blob/5fc2f6943bbeb3e12740681ba9aaa22d2bd8abf3/addons/sale_timesheet/models/product.py#L13-L17)